pallets/eth-bridge/src/lib.rs

```diff
@@ -229,10 +235,10 @@ pub mod pallet {
 
     #[pallet::storage]
     #[pallet::getter(fn active_ethereum_range)]
-    pub type ActiveEthereumRange<T: Config> = StorageValue<_, EthBlockRange, OptionQuery>;
+    pub type ActiveEthereumRange<T: Config> = StorageValue<_, ActiveEthRange, OptionQuery>;
 
     #[pallet::storage]
-    pub type DiscoveredEventsShard<T: Config> = StorageDoubleMap<
+    pub type DiscoveredEventsFractions<T: Config> = StorageDoubleMap<
```

Comment:
what is an Events Fraction? Is this a processing batch?

Reply:
effectively yes.
From the total of events found, it is broken down into fractions of the whole, then voted and processed individually.